opta/commands/apply.py

```diff
@@ -334,6 +326,39 @@ def _apply(
         )
 
 
+def _verify_semver(old_semver_string: str, current_semver_string: str) -> None:
+    if old_semver_string not in [DEV_VERSION, ""] and current_semver_string not in [
+        DEV_VERSION,
+        "",
+    ]:
+        old_semver = semver.VersionInfo.parse(old_semver_string)
+        current_semver = semver.VersionInfo.parse(current_semver_string)
+        if old_semver > current_semver:
+            raise UserErrors(
```

Comment:
We already have this test somewhere right? maybe we should remove that as well

Reply:
I don't think we do